packages/router/router.ts

```diff
@@ -3912,15 +3972,88 @@ async function loadLazyRouteModule(
     ...mapRouteProperties(routeToUpdate),
     lazy: undefined,
   });
+
+  return route;
```

Comment:
I forget if `route === routetoUpdate` here - if so we're good, otherwise we probably want to return `routeToUpdate` here?